mappings/net/minecraft/entity/LivingEntity.mapping

```diff
@@ -465,10 +479,14 @@ CLASS net/minecraft/unmapped/C_usxaxydn net/minecraft/entity/LivingEntity
 		ARG 1 target
 	METHOD m_scrnkbbn detectEquipmentUpdates ()V
 	METHOD m_sebydyvw endCombat ()V
+	METHOD m_sephwywk dropCustomLoot (Lnet/minecraft/unmapped/C_bdwnwhiu;Lnet/minecraft/unmapped/C_sbxfkpyv;Z)V
+		ARG 2 source
+		ARG 3 playerKill
```

Comment:
What is this?

Reply:
the third parameter? a boolean, whether the kill was by a player